images/extension-builders/tinygo/commands.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+# Copyright 2020 Tetrate
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+extension_build()  {
+	tinygo build -o "$1" -scheduler=none -target wasi main.go
+}
+
+extension_test()  {
+	exec go test -tags=proxytest -v ./...
+}
+
+extension_clean()  {
+	rm /source/extension.wasm || true
```

Comment:
[nit] This hardcoded file path invalidates configurability of build output file via Extension Descriptor.

Reply:
OK, this is not resolved yet. Let me take a look.

Reply:
the same problems seems to happen to Rust as well. .*wasm binaries placed at provided --build-file flag locations are not cleaned up in `cargo clean`. I'm going to simply remove this line since this seems useless.